New package: ShellStructureTopo v0.6.0

JuliaRegistrator opened the pull request:
- Registering package: ShellStructureTopo
- Repository: https://github.com/PetrKryslUCSD/ShellStructureTopo.jl
- Created by: @PetrKryslUCSD
- Version: v0.6.0
- Commit: 7bdcda81f23ea0c697719b7ff5f278a57254b600
- Reviewed by: @PetrKryslUCSD
- Reference: https://github.com/PetrKryslUCSD/ShellStructureTopo.jl/commit/7bdcda81f23ea0c697719b7ff5f278a57254b600#commitcomment-145787220
- Description: Detect topological entities on a triangular surface mesh.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6081d843fa465c10abd5c7f37a12c86f4458d3ed9a661a375e4987fb2b55c65180218ada8549becc3768885b1564059ae3021256397f061e2cbb6a6d5e20dc28eb84b8cc15514a2a66da506ecf0ffa39dc3886bea1a63a2997585c6b55fb389b848d470ae6af1e8d4515a1c9e990c5dc4c31e868dc338cd11cc68afa79325977c9d423cc34163e8b08b6e87c940c49c1678c4e593121eff85dbc3c2adbf2da9ddf830430ebc98503ccafcddfcc2fbb68b0f45bb0a7c206dea032ff79e747a5783 -->

goerz commented:
Could you add a little bit more documentation that would give some idea of how the package can be used?

JuliaRegistrator committed:
New package: ShellStructureTopo v0.6.0

UUID: 77fad3b7-27a3-4ff4-b61b-6ce40034767b
Repo: https://github.com/PetrKryslUCSD/ShellStructureTopo.jl.git
Tree: 6e0076394f6c82a7d97c6242b8b04656f4869270

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

PetrKryslUCSD commented:
[noblock]

PetrKryslUCSD commented:
[noblock]

goerz commented:
[noblock] you have to edit the existing https://github.com/JuliaRegistries/General/pull/113597#issuecomment-2308385798 to contain `[noblock]`
